<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendMessage</class>
 <widget class="QDialog" name="sendMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-11</x>
     <y>-11</y>
     <width>881</width>
     <height>951</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #d5f5e3,
        stop: 1 #d6eaf8
    );
    margin: 0; 
    padding: 0;
}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-40</y>
      <width>751</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 36pt &quot;Sylfaen&quot;;
    color: #34495e;
    border-radius: 10px;
    padding-top: 48px;
    margin-top:30px;
    font-size: 30px;
   
}
</string>
    </property>
    <property name="text">
     <string>Send a Message honestly without him knowing you</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="add_message">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>611</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTextEdit {
    min-width: 500px;
    max-width: 700px;
    heigth:350px;
    background-color: #f0f0f0;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}

QLineEdit::placeholder {
    color: #888;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Write a Message here ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_message">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>600</y>
      <width>432</width>
      <height>156</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border-radius: 8px;
    padding: 8px 16px;
    font-weight: bold;
    margin-bottom:100px;
	font: 18pt &quot;NSimSun&quot;;
    min-width: 400px;
    min-height: 40px;
    
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1c5980;
}
</string>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>710</y>
      <width>432</width>
      <height>156</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2ecc71;
    color: white;
    border-radius: 8px;
    padding: 8px 16px;
    font-weight: bold;
    margin-bottom:100px;
	font: 18pt &quot;NSimSun&quot;;
    min-width: 400px;
    min-height: 40px;
    
}

QPushButton:hover {
    background-color: #27ae60;
}

QPushButton:pressed {
    background-color: #1e8449;
}</string>
    </property>
    <property name="text">
     <string>Back to main window</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
	font: 18pt &quot;Sylfaen&quot;;
    color: #34495e;

</string>
    </property>
    <property name="text">
     <string>Show My name</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
	font: 14pt &quot;Sylfaen&quot;;
    color: #34495e;

</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
